firefox-ios/firefox-ios-tests/Tests/ClientTests/TrackingProtectionTests/CertificatesModelTests.swift

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+import X509
+@testable import Client
+
+final class CertificatesModelTests: XCTestCase {
```

Comment:
Can you please rename the file and class to `CertificatesViewModelTests`? It will be easier to find the tests for `CertificatesViewModel` in the future! ("View" is missing)

Great tests by the way, thanks!